src/UnderlineNav2/UnderlineNav.tsx

```diff
@@ -353,10 +353,15 @@ export const UnderlineNav = forwardRef(
                             {actionElementChildren}
 
                             {loadingCounters ? (
-                              <LoadingCounter />
+                              <Box as="span" data-component="counter">
+                                <LoadingCounter />
```

Comment:
non-blocking and asking out of ignorance: Is it necessary to add a span/wrapper to the `LoadingCounter` or should be hidden from screen readers? 🤔

Reply:
That wasn't an intentional change, sorry! Thanks for pointing this out.